autorest/azure/azure.go

```diff
@@ -22,6 +22,8 @@ import (
 	"io/ioutil"
 	"net/http"
 	"strconv"
+	"regexp"
+	"strings"
 
 	"github.com/Azure/go-autorest/autorest"
```

Comment:
I preffered this package since its azure specific but if you prefer the autorest package ( inside utility.go ) let me know